### PR TITLE
K8S-815: Run Prometheus Node Exporter on Etcd nodes

### PR DESCRIPTION
**Pull request by FestivalBobcats**

### CODE REVIEW

**Comment by FestivalBobcats** [comment]
Care to give this a look when you get a chance @lblackstone?

File: modules/ignition/node_exporter.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+data "ignition_systemd_unit" "node_exporter" {
```
Comment (lblackstone) [inline modules/ignition/node_exporter.tf]: This should go in `assets.tf` (same dir)

**Review by lblackstone — CHANGES_REQUESTED** [review]
Looks good AFAICT. Have you tested this change with kubernetes-installer?

**Comment by FestivalBobcats** [comment]
Alright, updated. And yep, I've been testing in the installer and have a PR for that repo as soon as this is merged.

**Review by lblackstone — APPROVED** [review]

**Pull request merged**